docs/API/core/notifications.md

```diff
@@ -1,5 +1,134 @@
 # Notifications
 
+### Check if the device has permissions to receive notifications
+
+Use the `Fliplet.Navigator.Notifications.hasPermission()` method to check whether the device has permissions from the user to receive notifications (including push notifications).
+
+```js
+Fliplet.Navigator.Notifications.hasPermission().then(function (hasPermission) {
+  if (hasPermission) {
+    // The user can receive notifications
+  }
+});
+```
+
+---
+
+### Check what type of permission the device has to receive notifications
+
+Use the `Fliplet.Navigator.Notifications.getPermission()` method to check whether what type of permission the device has been given to receive notifications. Possible values are `granted`, `denied` and `default`.
+
+```js
+Fliplet.Navigator.Notifications.getPermission().then(function(permission) {
+  switch (permission) {
+    case 'granted':
+      // the user has granted permissions to receive notifications
+      break;
+    case 'denied':
+      // the user has denied permissions to receive notifications
+      break;
+    case 'default':
+      // no specific choice has been made yet (e.g. not requested)
```

Comment:
In Chrome users can close the request without responding. In native apps user can kill the app before responding.

```suggestion
      // no specific choice has been made yet (e.g. not requested or user has not responded)
```